File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.py
class Solution:
    def maxProduct(self, nums: List[int]) -> int:
        max1_ind=0
        max2=float('-inf')
        for i in range(len(nums)):
            if(nums[i]>nums[max1_ind]):
                max1_ind=i
        for j in range(len(nums)):
            if(nums[j]>max2 and j!=max1_ind):
                max2=nums[j]
        return((nums[max1_ind]-1)*(max2-1))

File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.c
int maxProduct(int* nums, int numsSize) {
    int max1_ind=0;
        int max2=INT_MIN;
        for(int i=0;i<numsSize;i++)
        {
            if(nums[i]>nums[max1_ind])
            {
                max1_ind=i;
            }
        }
        for(int j=0;j<numsSize;j++)
        {
            if(nums[j]>max2 && j!=max1_ind)
            {
                max2=nums[j];
            }
        }
        return((nums[max1_ind]-1)*(max2-1));
}